staging/src/k8s.io/api/testdata/v1.19.0/certificates.k8s.io.v1.CertificateSigningRequest.after_roundtrip.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
These changes look wrong to me but they are what `UPDATE_COMPATIBILITY_FIXTURE_DATA=true go test k8s.io/api` auto generated.  I assume I have a mistake somewhere.